provider/maas/environ.go

```diff
@@ -1491,6 +1495,47 @@ func (environ *maasEnviron) StopInstances(ids ...instance.Id) error {
 
 }
 
+// Instances returns the instance.Instance objects corresponding to the given
+// slice of instance.Id.  The error is ErrNoInstances if no instances
+// were found.
+func (environ *maasEnviron) Instances(ids []instance.Id) ([]instance.Instance, error) {
```

Comment:
I take it this was just moving code?

Reply:
Yes; I should have isolated it from this PR/commit.